deflect/NetworkListener.cpp

```diff
@@ -40,48 +40,72 @@
 
 #include "NetworkListenerWorker.h"
 #include "PixelStreamDispatcher.h"
-
 #include "CommandHandler.h"
 
+#ifdef DEFLECT_USE_LUNCHBOX
+#  include <lunchbox/servus.h>
+#  include <boost/lexical_cast.hpp>
+#endif
+
 #include <QThread>
 #include <stdexcept>
 
 namespace deflect
 {
-
 const int NetworkListener::defaultPortNumber_ = 1701;
 
-NetworkListener::NetworkListener( int port)
-    : pixelStreamDispatcher_(new PixelStreamDispatcher())
-    , commandHandler_(new CommandHandler())
+namespace detail
+{
+class NetworkListener
+{
+public:
+    NetworkListener()
+#ifdef DEFLECT_USE_LUNCHBOX
+        : servus( "_displaycluster._tcp" )
```

Comment:
This string is defined at two places, shouldn't it be a shared static const like defaultPortNumber_ ?